packages/material-ui/src/Popover/Popover.js

```diff
@@ -560,19 +560,15 @@ Popover.propTypes = {
 };
 
 Popover.defaultProps = {
-  anchorReference: 'anchorEl',
   anchorOrigin: {
```

Comment:
Not so happy about fragmenting when default props are resolved. Any reason to keep some in `defaultProps`?

Reply:
No, I'm moving the all in this case.